content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Use Registries in Build Farm"
+date: 2020-12-15T16:08:01-04:00
+draft: false
+---
+
+The follow table shows the public DNS of each registry in build farm:
+
+| cluster | registry                                             | note                        |
+|---------|------------------------------------------------------|-----------------------------|
+| api.ci  | registry.svc.ci.openshift.org                        |                             |
+| app.ci  | registry.ci.openshift.org                            | the central CI registry     |
+| build01 | registry.build01.ci.openshift.org                    |                             |
+| build02 | registry.build02.ci.openshift.org                    |                             |
+| vsphere | registry.apps.build01-us-west-2.vmc.ci.openshift.org | only open to vsphere admins |
+
+# Login as a github user:
+
+* Ensure oc-cli has logged into the cluster: Copy the login command from [the console](/docs/getting-started/useful-links/#clusters).
+* Use oc token to log into the registry, e.g.,  `registry.ci.openshift.org`:
+
+```bash
+$ podman login -u $(oc whoami) -p $(oc whoami -t) --authfile auth.json registry.ci.openshift.org
+Login Succeeded!
+
+$ cat auth.json
+{
+	"auths": {
+		"registry.ci.openshift.org": {
+			"auth": "token"
+		}
+	}
+}
+```
+
+# Get token for a service account
+
+If some robot is required to log into the central CI registry, we need to create a service account: Choose a new namespace, and create a pull request to add a folder with the same name in [release/clusters/app.ci](https://github.com/openshift/release/tree/master/clusters/app.ci) containing the yaml files of the namespace, the service account, and its RBACs. The following `rolebinding` allows `sa/image-puller` in `namespace/ci` to pull all images in the namespace.
```

Comment:
Don't we have docs for this?

Reply:
Updated with the links of docs at upstream.
By we, you mean openshift-docs or dptp-docs?
I am adding this section for https://issues.redhat.com/browse/DPTP-1613?focusedCommentId=15503381&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-15503381